.travis.yml

```diff
@@ -1,46 +1,41 @@
-#Trying to go to the faster container infrastructure of github
+dist: trusty
 sudo: false
-
 language: php
 
 php:
-  - 5.6
-  - 7.0
   - 7.1
-  - nigthly
-  - hhvm
-
-matrix:
-  fast_finish: true
-  include:
-    - php: 5.6
-      env: deps=low
```

Comment:
It's here: https://github.com/doctrine/migrations/pull/537/files#diff-354f30a63fb0907d4ad57269548329e3R26